infrastructure/db/handler.js

```diff
@@ -1,253 +1,208 @@
-const { models, initDb } = require('./')
+const { models, getSequelize } = require('./')
 
 /**
 * Finds all the Folders of specific Team member
-* @param {int} teammemeberId id of the wanted teammember
+* @param {int} teammemberId id of the wanted teammember
 * @return {Promise} promise, with json response
 */
-exports.getFolders = async(teammemeberId) => {
-
-    return new Promise((resolve) => {
-
-        models.folder.findAll({ where : { teammemeberId : teammemeberId} }).then(folders => {
-
-            let resp = { folders : new Array() };
-            for(let i = 0; i < folders.length;i++) {
-                resp.folders[i] = {
-                    id : folders[i].id,
-                    label : folders[i].label
-                }
-            }
-            resolve(resp);
-
-        });
-
-    });
-}
+const getFolders = async(teammemberId) => {
+    try {
+        return await model.folder.findAll({ where : { teammemberId : teammemberId} });
+    } catch(err) {
+        throw Error(err);
+    }
+};
 
 /**
 * Adds folder to folders table with corresponding given information
 * @param {json} request post body request
-* @return Nothing
+* @return Nothing.
 */
-exports.addFolder = async(request) => {
-    models.folder.create({ label : request.label, teammemberId : request.id});
+const addFolder = async(request) => {
+    try {
+        models.folder.create({ label : request.label, teammemberId : request.id});
```

Comment:
Again, here also on line 23 you are not using await which won't wait for the function execution to finish.